template/template_test.go

```diff
@@ -350,19 +350,23 @@ func checkCVEBody(t *testing.T, body string, custom map[string]interface{}) {
 	t.Helper()
 	assert.True(t, strings.Contains(body, toString(custom["repo_url"])))
 	assert.True(t, strings.Contains(body, toString(custom["scanned_at"])))
+	assert.True(t, strings.Contains(body, toString(custom["total_dependencies"])))
+	dirDepArr := toArrMap(custom["direct_updates"])
+	assert.NotNil(t, dirDepArr)
+	transDepArr := toArrMap(custom["transitive_updates"])
+	assert.NotNil(t, transDepArr)
+
+	checkDepDataCVE(t, body, custom["direct_updates"])
+	checkDepDataCVE(t, body, custom["transitive_updates"])
+}
 
-	depArr := toArrMap(custom["vulnerable_deps"])
-	assert.NotNil(t, depArr)
-
-	for _, dep := range depArr {
+func checkDepDataCVE(t *testing.T, body string, deps interface{}) {
+	deps_ := toArrMap(deps)
+	for _, dep := range deps_ {
 		assert.True(t, strings.Contains(body, toString(dep["name"])))
-		cveArr := toArrMap(dep["cves"])
-		assert.NotNil(t, depArr)
-		for _, cve := range cveArr {
-			assert.True(t, strings.Contains(body, toString(cve["CVE"])))
-			cvss := fmt.Sprintf("[%s/10]", toString(cve["CVSS"]))
-			assert.True(t, strings.Contains(body, cvss))
-		}
+		assert.True(t, strings.Contains(body, toString(dep["version"])))
+		assert.True(t, strings.Contains(body, toString(dep["latest_version"])))
+		assert.True(t, strings.Contains(body, toString(dep["ecosystem"])))
```

Comment:
for CVE data there is no `ecosystem` .. and what about checking `CVE` and `CVSS` ??  Why you are moved validation check which was there and still applicable.